src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -56,16 +59,16 @@ def load_arguments(self, _):
             c.argument('default_consistency_level', arg_type=get_enum_type(DefaultConsistencyLevel), help="default consistency level of the Cosmos DB database account")
             c.argument('max_staleness_prefix', type=int, help="when used with Bounded Staleness consistency, this value represents the number of stale requests tolerated. Accepted range for this value is 1 - 2,147,483,647")
             c.argument('max_interval', type=int, help="when used with Bounded Staleness consistency, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 1 - 100")
-            c.argument('ip_range_filter', nargs='+', validator=validate_ip_range_filter, help="firewall support. Specifies the set of IP addresses or IP address ranges in CIDR form to be included as the allowed list of client IPs for a given database account. IP addresses/ranges must be comma-separated and must not contain any spaces")
+            c.argument('ip_rules', nargs='+', help="firewall support. Specifies the set of IP addresses or IP address ranges in CIDR form to be included as the allowed list of client IPs for a given database account. This should be an array of ipRanges or addresses", is_preview=True)
```

Comment:
Parameter name change will break existing users. What about using options_list to provide multiple parameter names?

Reply:
Reverted to the previous paramter name, internally changed the implementation